common/test/acceptance/pages/lms/fields.py

```diff
@@ -182,6 +182,7 @@ def get_non_editable_mode_value(self, field_id):
         Return value of field in `display` or `placeholder` mode.
         """
         self.wait_for_field(field_id)
+        self.wait_for_ajax()
```

Comment:
I think it would be better for the test to wait for the expected value, instead of putting wait_for_ajax in here. We have found that wait_for_ajax is too general and is likely to lead to sporadic failures.